### PR TITLE
Add v1.0 draft

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: _posts/2018-10-23-version-1.0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+protocols (like async-await, concurrent.futures, Queues, Locks, and so on).
+Additionally, Dask is well used in production today across sectors ranging from
+risk-tolerant industries like startups and quantitative finance shops, to
+risk-averse institutions like banks, large enterprise, and governments.
+
+*When we say that Dask has reached 1.0 we mean that it is ready to be used in
+production.  We are late in saying this.  This happened a long time ago.*
+
+
+But there will still be bugs, deprecations, and so on
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]: This section title could be reworded to "Development will continue", or something similar, to sound a little more optimistic than "there will still be bugs". Although, there indeed will still be bugs.

```suggestion
Development will continue
```

File: _posts/2018-10-23-version-1.0.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+It is common in the PyData ecosystem to wait a *long time* before releasing a
+version 1.0.  For example neither Pandas nor Scikit-Learn, arguably two of the
+most well used PyData packages in production, have yet declared a 1.0 version
+number (today they are at versions 0.23 and 0.20 respectively).  And yet each
+package is widely used in production by organizations that demand high degrees
+of stability.
+
+Dask is not as API-stable as Pandas or Scikit-Learn, but it's pretty close.
+The project rarely invents new APIs, instead preferring to implement
+pre-existing APIs (like the Numpy/Pandas/Scikit-Learn APIs) or standard language
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]:
```suggestion
pre-existing APIs (like the NumPy/Pandas/Scikit-Learn APIs) or standard language
```

File: _posts/2018-10-23-version-1.0.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Additionally, Dask is well used in production today across sectors ranging from
+risk-tolerant industries like startups and quantitative finance shops, to
+risk-averse institutions like banks, large enterprise, and governments.
+
+*When we say that Dask has reached 1.0 we mean that it is ready to be used in
+production.  We are late in saying this.  This happened a long time ago.*
+
+
+But there will still be bugs, deprecations, and so on
+-----------------------------------------------------
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]: Goes along with the section title suggestion 

```suggestion
--------------------------
```

File: _posts/2018-10-23-version-1.0.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Dask Version 1.0
+author: Dask Team
+tags: [Programming, Python, scipy, dask]
+theme: twitter
+---
+{% include JB/setup %}
+
+We are pleased to announce the release of Dask version 1.0.0.
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]:
```suggestion
We are pleased to announce the release of Dask version 1.0.0!
```

I'm excited about `1.0`!

File: _posts/2018-10-23-version-1.0.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+Usually in release blogposts we outline important features and changes since
+the last major version.  Because of the 1.0 version number, this post will be a
+bit different.  Instead we'll talk about what this version number means to us,
+and discuss the broader context of Dask projects more generally.
+
+
+What 1.0 means to us
+--------------------
+
+The software version of 1.0 means different things to different groups.
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]: Slight rewording

```suggestion
Version 1.0 software means different things to different groups.
```

File: _posts/2018-10-23-version-1.0.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+package is widely used in production by organizations that demand high degrees
+of stability.
+
+Dask is not as API-stable as Pandas or Scikit-Learn, but it's pretty close.
+The project rarely invents new APIs, instead preferring to implement
+pre-existing APIs (like the Numpy/Pandas/Scikit-Learn APIs) or standard language
+protocols (like async-await, concurrent.futures, Queues, Locks, and so on).
+Additionally, Dask is well used in production today across sectors ranging from
+risk-tolerant industries like startups and quantitative finance shops, to
+risk-averse institutions like banks, large enterprise, and governments.
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]: Not sure about this one, but "enterprises" sounds right to me 

```suggestion
risk-averse institutions like banks, large enterprises, and governments.
```

**Review by jrbourbeau — COMMENTED** [review]
Thanks for writing this up @mrocklin, overall this post looks really good to me

I've included a few minor, slightly nitpicky comments. I'm trying out the new "Suggested changes" GitHub feature, so hopefully you'll be able to easily pick and choose which review comments you want to include.

**Comment by mrocklin** [comment]
Thanks!  That was fun.  I still need to try out that interface on the recommender's side.

File: _posts/2018-10-23-version-1.0.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+Additionally, Dask is well used in production today across sectors ranging from
+risk-tolerant industries like startups and quantitative finance shops, to
+risk-averse institutions like banks, large enterprises, and governments.
+
+*When we say that Dask has reached 1.0 we mean that it is ready to be used in
+production.  We are late in saying this.  This happened a long time ago.*
+
+
+Development will continue
+--------------------------
```
Comment (jrbourbeau) [inline _posts/2018-10-23-version-1.0.md]: My mistake, there's an extra `-`

```suggestion
-------------------------
```

**Comment by mrocklin** [comment]
Merging in one hour.

**Pull request merged**